can/io/sqlite.py

```diff
@@ -128,7 +128,9 @@ class SqliteWriter(MessageWriter, BufferedReader):
     MAX_BUFFER_SIZE_BEFORE_WRITES = 500
     """Maximum number of messages to buffer before writing to the database"""
 
-    def __init__(self, file: StringPathLike, table_name: str = "messages") -> None:
+    def __init__(
+        self, file: StringPathLike, table_name: str = "messages", **options: Any
```

Comment:
```suggestion
        self, file: StringPathLike, table_name: str = "messages", **kwargs: Any
```

This is usually called `kwargs` in python projects

Reply:
> Also: Shouldn't there be an exception when the user demands "append" but the logger does not support it?

I think this is brilliant.